spec.md

```diff
@@ -190,18 +190,28 @@ within the same JSON object.
 ### source
 
 - Type: `URI-reference`
-- Description: This describes the event producer. Often this will include
-  information such as the type of the event source, the organization publishing
-  the event, the process that produced the event, and some unique identifiers.
-  The exact syntax and semantics behind the data encoded in the URI is event
-  producer defined.
+- Description: Identifies the event producer. Often this will include
+  information such as the type of the event source, the organization
+  publishing the event, the process that produced the event, and some unique
+  identifiers. The exact syntax and semantics behind the data encoded in the URI
+  is event producer defined.
+
+  An application MUST assign a distinct `source` to each distinct producer.
```

Comment:
On the call today (25.04.2019) we accepted the terminology in PR #420 for Source, Producer etc. This line contradicts that terminology, because a producer is defined as a specific instance creating events. Each logical system in which the occurrences are recorded/created ("Source" in the terminology), causing events, should have a distinct `source` attribute.

For example, two instances of a "Users service" would be distinct producers, but they *should* still have the same `source` field. Together they are a source and have a distinct `source` attribute compared to another distinct source, say a "Routing service".

Oddly enough, IIUC it should say:
> An application MUST assign a distinct `source` to each distinct source.

Writing this out, I realise this is as much a problem with the new terminology clashing with the attribute name and description. I am leaving a comment on #420 saying as much, and @duglin agreed to not merge it I think.

Reply:
Fixed in next update - will make more sense when #420 is applied.